google/services/cloudrunv2/resource_cloud_run_v2_service.go

```diff
@@ -537,12 +537,12 @@ If not specified or 0, defaults to 80 when requested CPU >= 1 and defaults to 1
 								Schema: map[string]*schema.Schema{
 									"max_instance_count": {
 										Type:        schema.TypeInt,
-										Optional:    true,
+										Required:    true,
```

Comment:
From what I understand of https://github.com/hashicorp/terraform-provider-google/issues/18969#issuecomment-2269519946
We need to use `RequiredWith`, not `Required` here